assets/src/edit-story/editorApp.js

```diff
@@ -70,14 +71,16 @@ function App({ config }) {
                                 <AutoSaveHandler />
                                 <TransformProvider>
                                   <DropTargetsProvider>
-                                    <GlobalStyle />
-                                    <DevTools />
-                                    <DefaultMoveableGlobalStyle />
-                                    <CropMoveableGlobalStyle />
-                                    <ModalGlobalStyle />
-                                    <CalendarStyle />
-                                    <KeyboardOnlyOutlines />
-                                    <Layout />
+                                    <HelpCenterProvider>
```

Comment:
We should be good here with updates since this is using useContextSelector under the hood, but something to keep an eye on cus any app wide rerenders could cause some jank in the transitions

Reply:
I was thinking about your [post about stores](https://formidable.com/blog/2021/stores-no-context-api/) while writing this lol